Sofa/Component/Topology/Container/Dynamic/src/sofa/component/topology/container/dynamic/EdgeSetTopologyModifier.cpp

```diff
@@ -50,6 +50,13 @@ void EdgeSetTopologyModifier::init()
 {
     PointSetTopologyModifier::init();
     getContext()->get(m_container);
+
+    if(!m_container)
+    {
+        msg_error() << "EdgeSetTopologyContainer not found in context";
```

Comment:
Does "context" is enough clear for users? Or maybe "current Node" / same Node as this->getName()